test/test proj/android.toolchain.cmake

```diff
@@ -0,0 +1,726 @@
+# Copyright (C) 2016 The Android Open Source Project
```

Comment:
Sure, but I took a real world example of such a file. Any particular reason why would you want a different one?

Reply:
I just thought that if we have custom toolchain files for tests, we will be able to extend these files when we will support new features (add new cmake options, cases and etc.).
Moreover we can create a set of toolchain files in order to validate different cases and don't mix such cases.

Reply:
That sounds good, but I'm not sure if that will be necessary in the future, as far as I recall `CMake 3.20` will contain the `Android` toolchains within the `cmake` binary, so this will be unnecessary for newer versions.

But I recommend we create custom ones when we add new features.

I would like to leave this `toolchain` since the most often use case is to take the official `Android` toolchain file and use that.